libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -9,15 +9,37 @@
         {{ b.authors.join(',') }}
       </div>
     </div>
-    <div>
-      <button
+    <div *ngIf="b.finished" data-testing="finished-date" tabindex="0"
+         [attr.aria-label]="'Finished on' + b.finishedDate" class="reading-list-item--details--author">
+            Finished on: {{ b.finishedDate }}
+    </div>
```

Comment:
Also, the result here in the UI looks a bit too crammed. A user will not be able to see which book they finished.. Could we add the finished date and text to some other place, eg. below the title, or on the side, or someplace else, to that at least a user is still able to see the title of the book and the authors?
![Screenshot 2021-06-11 at 7 18 02 PM](https://user-images.githubusercontent.com/6603745/121717353-c1a5ee80-cae9-11eb-94d6-a42587d6a7da.png)

Reply:
<img width="629" alt="Screen Shot 2021-06-28 at 6 49 04 PM" src="https://user-images.githubusercontent.com/80428520/123724656-8d4c6500-d841-11eb-80b5-48f6cc334442.png">
UI is corrected to show Title, author name and date format